pydantic/fields.py

```diff
@@ -793,6 +785,10 @@ def Field(  # noqa: C901
     if validation_alias in (_Unset, None):
         validation_alias = alias
 
+    include = extra.pop('include', None)  # type: ignore
+    if include is not None:
+        warn('`include` is deprecated and does nothing. It will be removed, use `exclude` instead', DeprecationWarning)
```

Comment:
AFAIU, this is not a deprecation... If the functionality is removed, it's a removal. The parameter itself is deprecated.

Also, how is `exclude` a replacement for `include`? 🤔

Reply:
> AFAIU, this is not a deprecation... If the functionality is removed, it's a removal. The parameter itself is deprecated.
> 

I think it is a deprecation as it did nothing before and still does nothing. So, no change in functionality. just warning the user.
 
> Also, how is `exclude` a replacement for `include`? 🤔

`include=False` can be replaced with `exclude=True`. That's why I suggested using `exclude`.